docs/src/dev-manual/archivematica.md

```diff
@@ -76,6 +80,9 @@ AM API and SFTP configuration:
     sftp_user=archivematica
     sftp_remote_dir=/enduro_transfers
     sftp_private_key_passphrase=
+    amss_url=http://host.k3d.internal:62081
+    amss_user=test
+    amss_api_key=secret
```

Comment:
If we add the `amss_location_id` to the secret and use it in the environment for the enduro services and the create location job, we could be sure they match in this environment.